### DISCUSSION TITLE
ERROR: Horizontal Rule Generates a Quoted Block

### DISCUSSION THREAD

**Discussion by rickgordon** [protz/thunderbird-conversations#2060] [General]
[ Thunderbird 115.x / Mac Monterey ]

An email containing the following HTML …

&lt;!DOCTYPE html&gt;
&lt;html&gt;
  &lt;head&gt;
    &lt;meta http-equiv="content-type" content="text/html; charset=UTF-8"&gt;
  &lt;/head&gt;
  &lt;body&gt;
    &lt;p&gt;test&lt;/p&gt;
    &lt;hr&gt;
    &lt;p&gt;test&lt;/p&gt;
  &lt;/body&gt;
&lt;/html&gt;

<!DOCTYPE html>
<html>
  <head>
    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
  </head>
  <body>
    <p>test</p>
    <hr>
    <p>test</p>
  </body>
</html>

… will render in Thunderbird Conversations in such a way that the `<hr>` will not show, and the line where the ``<hr>` should have been and all the text below it will render like a quote, with a blue vertical line and indentation, like the example below:

![horizontal-rule-bug-example](https://github.com/thunderbird-conversations/thunderbird-conversations/assets/1865905/13cccd28-2d09-4375-9f20-e10f7c5af0c0)

This does not occur with Thunderbird's rendering outside of Thunderbird Conversations.

**Comment by Standard8** [comment]
This has been filed as https://github.com/thunderbird-conversations/thunderbird-conversations/issues/2061
